core/lib/generators/solidus/install/install_generator.rb

```diff
@@ -322,18 +323,27 @@ def detect_payment_method_to_install
       return 'paypal' if Bundler.locked_gems.dependencies['solidus_paypal_commerce_platform']
       return 'bolt' if Bundler.locked_gems.dependencies['solidus_bolt']
 
-      options[:payment_method] ||
-        (options[:auto_accept] && @selected_frontend == 'classic' ? 'paypal' : 'none') ||
-        (@selected_frontend != 'classic' && 'none') || # bail out if it's not classic
+      descriptions = {
+        paypal: "- [#{set_color 'paypal', :bold}] Install `solidus_paypal_commerce_platform` (#{set_color :default, :bold}).",
+        bolt: "- [#{set_color 'bolt', :bold}] Install `solidus_bolt`.",
+        none: "- [#{set_color 'none', :bold}] Skip installing a payment method.",
```

Comment:
```suggestion
        none: "- [#{set_color 'none', :none}] Skip installing a payment method.",
```

Reply:
Not sure I understand, would this be just to not emphasize the `none` option? or do we have a default color for `:none` (vs. not using `set_color` at all).

Reply:
Haha, sorry, I mixed `bold` with `bolt` in my head 😓

Reply:
Hahah, makes sense, all good then 🙌